dockerbuild/Dockerfile-backend

```diff
@@ -1,10 +1,13 @@
 FROM openjdk:8
 
 EXPOSE 6543
-
+ 
 # Maven installs openjdk-7 as the default Java. update-alternatives points java back to 8.
 RUN apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv 0C49F3730359A14518585931BC711F9BA15703C6 && \
-    echo "deb [ arch=amd64 ] http://repo.mongodb.org/apt/ubuntu trusty/mongodb-org/3.4 multiverse" | tee /etc/apt/sources.list.d/mongodb-org-3.4.list && \
+    echo 'deb http://httpredir.debian.org/debian jessie-backports main' > \
+    /etc/apt/sources.list.d/backports.list &&
```

Comment:
@zitterbewegung Needs a trailing \ here and the next line. Once that's fixed, this PR will be good to merge.